src/Payments/Api.php

```diff
@@ -174,4 +175,58 @@ public function process_legacy_payment( PaymentContext $context, PaymentResult &
 		$result->set_payment_details( array_merge( $result->payment_details, $gateway_result ) );
 		$result->set_redirect_url( $gateway_result['redirect'] );
 	}
+
+	/**
+	 * Verify all dependencies of registered payment methods have been registered.
+	 * If not, remove that payment method script from the list of dependencies
+	 * of Cart and Checkout block scripts so it doesn't break the blocks and show
+	 * an error in the admin.
+	 */
+	public function verify_payment_methods_dependencies() {
+		global $wp_scripts;
```

Comment:
I think we can avoid the global by calling `$wp_scripts = wp_scripts();`?

Reply:
Yup, much better. Changed in 5d46dbedbaf3f1e5eadee2070656ebb4aed18a40.